data.json

```diff
@@ -1349,4 +1349,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/51391473/134031514-b0bbff29-3974-4f05-a2c6-a5c163e90216.png",
     "issueNumber": 726
   }
+  {
+    "name": "Aniket Kumar",
+    "githubUsername": "aniket-24",
+    "imageUrl": "https://user-images.githubusercontent.com/76562459/133885424-2f4af13d-239f-4e59-a2e7-e6324e275e55.png",
```

Comment:
I would recommend more of an image like this.
![image](https://user-images.githubusercontent.com/51391473/134790539-719a671c-65d8-42b4-8ea7-3c2cf6f760bf.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/134790539-719a671c-65d8-42b4-8ea7-3c2cf6f760bf.png",
```